entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
If the first three lines of this file are no longer needed, it would be great to not have this file anymore so the README preview on GitHub doesn't get pushed down *too* much farther down the page.

Reply:
Nevermind I see what you mean now. Ill move the files to another directory

Reply:
I meant something different but this works just as well. I meant to say the first three lines excluding the shebang (so lines 1-4). I was thinking that if you're able to put the compiled files and mounted files in two different directories, then `entrypoint.sh` could be reduced to simply:

```bash
#!/bin/bash
npx concurrently "venv/bin/mkdocs serve -a 0.0.0.0:8000" "npm:dev -- --host 0.0.0.0 --port 5173"
exec "$@"
```

at which point you could delete `entrypoint.sh` and use `ENTRYPOINT` to run the command e.g.
```dockerfile
ENTRYPOINT npx concurrently "venv/bin/mkdocs serve -a 0.0.0.0:8000" "npm:dev -- --host 0.0.0.0 --port 5173"
```